src/actions/remove_token.ts

```diff
@@ -0,0 +1,20 @@
+import { token, store } from '../utils/constants';
+
+if (token) {
```

Comment:
I don't think @websitedesignanddeveloper that we need to check if there is a token or not. Because this script will only be shown if there is a token so we can take that as a fact and remove this check.

Reply:
Done.